### PR TITLE
feat: notifications

### PR DESCRIPTION
**Pull request by shoom3301**
1. "Notification template" collection
2. "Notification" collection
3. "Telegram subscription" collection

### CODE REVIEW

File: src/api/notification-template/content-types/notification-template/schema.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "kind": "collectionType",
+  "collectionName": "notification_templates",
+  "info": {
+    "singularName": "notification-template",
+    "pluralName": "notification-templates",
+    "displayName": "Notification template",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
```
Comment (anxolin) [inline src/api/notification-template/content-types/notification-template/schema.json:11]: this is the flag I mentioned. I would set it to false

File: src/api/notification-template/content-types/notification-template/schema.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "type": "richtext"
+    },
+    "url": {
+      "type": "string"
+    },
+    "push": {
+      "type": "boolean"
+    },
+    "thumbnail": {
+      "type": "string"
```
Comment (anxolin) [inline src/api/notification-template/content-types/notification-template/schema.json:28]: Let\s change it to Media, only 1 item, and required field. I believe in the cloud will work

File: src/api/notification-template/content-types/notification-template/schema.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "displayName": "Notification template",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "title": {
+      "type": "string"
```
Comment (anxolin) [inline src/api/notification-template/content-types/notification-template/schema.json:16]: should be mandatory

File: src/api/notification-template/content-types/notification-template/schema.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    "description": {
+      "type": "richtext"
```
Comment (anxolin) [inline src/api/notification-template/content-types/notification-template/schema.json:19]: should be mandatory

File: src/api/notification-template/content-types/notification-template/schema.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    "url": {
+      "type": "string"
+    },
+    "push": {
+      "type": "boolean"
```
Comment (anxolin) [inline src/api/notification-template/content-types/notification-template/schema.json:25]: What about putting it as mandatory, default FALSE

File: src/api/telegram-subscription/content-types/telegram-subscription/schema.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
+  "kind": "collectionType",
+  "collectionName": "telegram_subscriptions",
+  "info": {
+    "singularName": "telegram-subscription",
+    "pluralName": "telegram-subscriptions",
+    "displayName": "Telegram subscription"
+  },
+  "options": {
+    "draftAndPublish": true
```
Comment (anxolin) [inline src/api/telegram-subscription/content-types/telegram-subscription/schema.json:10]: Safe, I think this should be false

File: src/api/notification/content-types/notification/schema.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    "singularName": "notification",
+    "pluralName": "notifications",
+    "displayName": "Notification",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
```
Comment (anxolin) [inline src/api/notification/content-types/notification/schema.json:14]: should we have a field consumed? or you want to DELETE when consumed?

**Review by anxolin — APPROVED** [review]
Approved with comments

**Comment by anxolin** [comment]
@shoom3301  This and other PRs during the hackathong. Should we go back to the comments and revisit them?

File: src/api/notification-template/content-types/notification-template/schema.json (hunk 1 above)
Comment (shoom3301) [reply src/api/notification-template/content-types/notification-template/schema.json:11]: Fixed

File: src/api/notification-template/content-types/notification-template/schema.json (hunk 3 above)
Comment (shoom3301) [reply src/api/notification-template/content-types/notification-template/schema.json:16]: Fixed

File: src/api/notification-template/content-types/notification-template/schema.json (hunk 4 above)
Comment (shoom3301) [reply src/api/notification-template/content-types/notification-template/schema.json:19]: Fixed

File: src/api/notification-template/content-types/notification-template/schema.json (hunk 5 above)
Comment (shoom3301) [reply src/api/notification-template/content-types/notification-template/schema.json:25]: Fixed

File: src/api/telegram-subscription/content-types/telegram-subscription/schema.json (hunk 6 above)
Comment (shoom3301) [reply src/api/telegram-subscription/content-types/telegram-subscription/schema.json:11]: Fixed

File: src/api/notification/content-types/notification/schema.json (hunk 7 above)
Comment (shoom3301) [reply src/api/notification/content-types/notification/schema.json:14]: We decided to use `lastConsumedNotificationDate` value

**Pull request merged**